deploy/packages/bbp-packages.yaml

```diff
@@ -39,7 +39,8 @@ packages:
       - compiler
       - python
     specs:
-      - py-simwriter@0.4.3 ^py-rpy2@2.8.6 ^openjdk
+      - py-bbp-analysis-framework@0.6.39 ^py-rpy2@2.8.6
```

Comment:
Maybe we should add this `py-rpy2` to `python-packages`. Not sure why we have to call into R…

Reply:
It's coming from `https://github.com/NeuralEnsemble/NeuroTools/blob/master/requirements.txt`.

Reply:
I think it is not used. I'll try to drop that dependency...

Reply:
Thanks. Seeing that there just piqued my interest…